source/globalCommands.py

```diff
@@ -1861,6 +1861,7 @@ def script_review_currentCharacter(self, gesture: inputCore.InputGesture):
 			if c is not None:
 				speech.speakMessage("%d," % c)
 				speech.speakSpelling(hex(c))
+				braille.handler.message(f"{c}, {hex(c)}")

```

Comment:
should this be a translatable message given the use of comma? I know the previous speech example isn't but perhaps should be as well?